.github/workflows/canary-release.yml

```diff
@@ -17,12 +17,13 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+
       - name: Setup Git User
         uses: fregante/setup-git-user@v2
-      - name: Set up pnpm 8.x
+
+      - name: Set up pnpm
         uses: pnpm/action-setup@v2
-        with:
-          version: 8
```

Comment:
The action will [automatically pick the correct version](https://github.com/pnpm/action-setup?tab=readme-ov-file#version) using the [`packageManager` value](https://github.com/makeswift/makeswift/pull/937/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L3) in the `package.json`, no need to duplicate it here.